### PR TITLE
Adding access keys to nearlib

### PR DESCRIPTION
**Pull request by evgenykuzyakov**
Adding a way to add access keys in nearlib.
Adding a test to add an access key and call a contract using an access key.
For #687

### CODE REVIEW

File: nearlib/package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "nearlib",
   "description": "Javascript library to interact with near blockchain",
-  "version": "0.5.4",
+  "version": "0.5.5",
```
Comment (vgrichina) [inline nearlib/package.json]: @evgenykuzyakov isn't this change breaking API? Means this should be version 0.6.0, see http://semver.org

File: nearlib/account.js [hunk 2]
```diff
@@ -1,6 +1,6 @@
 const bs58 = require('bs58');
 
-const { CreateAccountTransaction, SignedTransaction } = require('./protos');
+const { google, AccessKey, AddKeyTransaction, CreateAccountTransaction, SignedTransaction } = require('./protos');
```
Comment (janedegtiareva) [inline nearlib/account.js]: do we need to add the new classes to index.js and browser.js?

File: nearlib/test/integration.test.js [hunk 3]
```diff
@@ -167,6 +167,81 @@ test('create account with a new key and then view account returns the created ac
...
+    });
+
+    test('make function calls using access key', async () => {
+        // Adding access key
+        const keyForAccessKey = await KeyPair.fromRandomSeed();
+        const addAccessKeyResponse = await account.addAccessKey(
+            newAccountId,
+            keyForAccessKey.getPublicKey(),
+            contractId,
+            '',
```
Comment (vgrichina) [inline nearlib/test/integration.test.js]: no idea what are `''`, `''`, `0` here without going to check `addAccessKey` method.

I think it makes sense to either pass parameters as part of options object, or save them to named variables before passing (unless immediately obvious what it is)

File: nearlib/test/integration.test.js [hunk 4]
```diff
@@ -167,6 +167,81 @@ test('create account with a new key and then view account returns the created ac
...
+        await nearjs.waitForTransactionResult(addAccessKeyResponse);
+        // Replacing public key for the account with the access key
+        await keyStore.setKey(newAccountId, keyForAccessKey);
+        // test that load contract works and we can make calls afterwards
+        const contract = await nearjs.loadContract(contractId, {
+            viewMethods: ['hello', 'getValue'],
+            changeMethods: ['setValue'],
+            sender: newAccountId,
+        });
+        const helloResult = await contract.hello({ name: 'trex' });
```
Comment (vgrichina) [inline nearlib/test/integration.test.js]: looks like `contract.hello` call isn't necessary for test

also I think at this point it makes sense to extract "test that we can write value and read back" into a helper function (cause it's useful for multiple tests)

Comment (evgenykuzyakov) [reply nearlib/test/integration.test.js]: Yeah. I guess view calls are not needed, unless we change a logic.

File: nearlib/test/integration.test.js (hunk 3 above)
Comment (evgenykuzyakov) [reply nearlib/test/integration.test.js]: My IDE shows which argument it is for unnamed args. I'll add comments

File: nearlib/account.js (hunk 2 above)
Comment (evgenykuzyakov) [reply nearlib/account.js]: No, since we don't return them to the browser or index

File: nearlib/package.json (hunk 1 above)
Comment (evgenykuzyakov) [reply nearlib/package.json]: It's non-breaking change, but it does require certain version of nearcore that has proto changes.

Comment (evgenykuzyakov) [reply nearlib/package.json]: But it's not backward compatible, so bumping to 0.6.0.
Not going to publish it until we add the new signer

File: nearlib/test/integration.test.js (hunk 3 above)
Comment (vgrichina) [reply nearlib/test/integration.test.js]: @evgenykuzyakov IDE does this, but e.g. right here we don't have this feature available. It's not like most time _reading_ code is spent inside of IDE.

**Pull request merged**